src/main/resources/init_db.sql

```diff
@@ -23,7 +23,7 @@ CREATE TABLE `cars` (
                         `id` BIGINT NOT NULL AUTO_INCREMENT,
                         `model` VARCHAR(255) NOT NULL,
                         `is_deleted` TINYINT NOT NULL DEFAULT 0,
-                        `manufacturer_id` carsINT NOT NULL,
+                        `manufacturer_id` INT NOT NULL,
```

Comment:
but `manufacturers.id` has type `BIGINT(11)`, 
so `manufacturer_id` also should be of type `BIGINT(11)`